### ISSUE TITLE
styled-components/macro doesn't work in latest version of package

### ISSUE THREAD

**Issue by kamesdev** [styled-components/styled-components#3635]
Version 5.2.1 and above

**Comment by razvan-soare** [comment]
Macro is not working anymore confirm. @5.3.3

**Comment by TiE23** [comment]
Actually, I ask anyone to attempt to start a new create-react-app project on their machine. Install `styled-components`, `babel-plugin-styled-components@1.13.3`, and `babel-plugin-macro`. Then define a basic styled-component while importing from "_styled-components/macro_" and see if it works for you. I tried with both CRA vanilla and CRA typescript and **neither** worked.

**Comment by esetnik** [comment]
I also was able to reproduce on codesandbox in #3645 and I have a workaround solution implemented in #3647 so please go upvote that and let's get it merged.

**Comment by agriffis** [comment]
@probablyup This issue was fixed by release v2.0.3 of the babel plugin, so it can be closed now.

**Comment by agriffis** [comment]
@TiE23 You posted a sandbox earlier in this issue. Here it is with v2.0.2 (prior to the fix) and then with v2.0.6 (after the fix that landed in v2.0.3)

### v2.0.2 (broken)
![image](https://user-images.githubusercontent.com/50637/158206006-4934532a-8cbe-484c-b9a7-7d9984caefc2.png)

### v2.0.6 (fixed)

![image](https://user-images.githubusercontent.com/50637/158205598-9ddf9e2f-ee62-4a76-abc3-068dccba7677.png)

Of course this doesn't necessarily mean that all bugs are fixed, but this specific issue seems to be fixed.

I see your most recent comment showing combinations that aren't working for you. Is that something you're able to share in a sandbox or provide more details on how it's failing?

**Comment by TiE23** [comment]
@agriffis I have to apologize - I just triple checked my results and for me it's working now.

Don't know what happened. Currently only have one terminal open to run my client - forcing me to close the client process before updating any package and then re-launching the client. First time I ran with the new plugin v2.0.6 it didn't work. I then tried SC v5.2.1, which lead me to write my reply.

Re-installing SC v5.3.3 and launching my client I'm very happy to see it working.

Thank you for replying so fast. It also made me blush. I feel embarrassed because I know my tone must've sounded whiny and entitled.

![github-my-mistake](https://user-images.githubusercontent.com/2433505/158209682-86fc37cd-5bb1-4a8b-9e9a-cf3154fcbe6a.png)
